docs/whats_new.rst

```diff
@@ -27,7 +27,9 @@ Enhancements
 
 Bugs
 ~~~~
-- 
+- amplitude gradients are correctly computed for layers with multiple filters
+  (before, they were accidentally summed over all previous filters in the layer) (:gh:`167` by `Robin Tibor Schirrmeister`_)
+- get_output_shape and compute_amplitude_gradients assume 3d, not 4d inputs (:gh:`166` by `Robin Tibor Schirrmeister`_)
```

Comment:
is get_output_shape and compute_amplitude_gradients part of the public API? if so it should be added to https://braindecode.org/api.html and then here you should be able to use :func:`...` to have a link in the what's new page.